src/node_options.cc

```diff
@@ -122,6 +122,11 @@ void EnvironmentOptions::CheckOptions(std::vector<std::string>* errors) {
                       "--experimental-modules be enabled");
   }
 
+  if (experimental_wasi_modules && !experimental_wasm_modules) {
+    errors->push_back("--experimental-wasi-modules requires "
+                      "--experimental-wasm-modules be enabled");
+  }
```

Comment:
Alternatively, could we maybe `Implies("--experimental-wasi-modules", "--experimental-wasm-modules");`? Having to pass both flags seems a bit much…

Reply:
since it also allows modules that don't conform to wasi, i thought it best to have to pass both.

Reply:
@devsnek Yeah … just seems like a lot to type :)

Reply:
hasn't been a huge problem for me but i'm willing to change it

Reply:
I'd vote for the automatic implication as well.